extra/generic-init.d/celeryd

```diff
@@ -284,7 +284,7 @@ restart_workers () {
 
 
 kill_workers() {
-    _chuid kill $CELERYD_NODES --pidfile="$CELERYD_PID_FILE"
+    _chuid kill $DAEMON_OPTS $CELERYD_NODES --pidfile="$CELERYD_PID_FILE"
```

Comment:
Minor, but just for uniformity with the other commands (see `restart_workers` function above) can you place `$DAEMON_OPTS` after `$CELERYD_NODES`?

Reply:
done.